fuse/server.go

```diff
@@ -64,8 +61,12 @@ func (ms *Server) SetDebug(dbg bool) {
 	ms.opts.Debug = dbg
 }
 
-func (ms *Server) Version() (major, minor uint32) {
-	return ms.versionMinor, ms.versionMinor
+func (ms *Server) SupportsNotify(op int) bool {
+	switch op {
+	case NOTIFY_INVAL_ENTRY:
+		return ms.kernelSettings.Major >= 7 && ms.kernelSettings.Minor >= 12
```

Comment:
confirming the version when support was added:
https://github.com/osxfuse/kext/blob/5639d3e9c9aa32c532a0faef562d6c459a485f8d/osxfuse/fuse_ipc.h#L511